docker/aeternity_node_mean16.yaml

```diff
@@ -20,7 +20,7 @@ chain:
   persist: false
   hard_forks:
     "1": 0
-    "6": 1
+    "5": 1
```

Comment:
Otherwise tests won't work 🤷‍♀️

Reply:
any idea how to isolate the tests? 🤔

Reply:
After doing complete compatibility with Ceres I will create a separate PR to test sdk with it, like https://github.com/aeternity/aepp-sdk-js/pull/1808